### PR TITLE
enable/disable tasks actions depending on selection

### PR DESCRIPTION
**Pull request by oroulet**

### CODE REVIEW

**Comment by oroulet** [comment]
please use rebase and merge to merge. That will make my life easier :-)

**Comment by PKEv** [comment]
Are you sure that these operations need to ban for multiple tasks?

**Comment by oroulet** [comment]
Need to be banned when no task is selected! But you can check and enable
those that support multiple tasks

File: qtodotxt/ui/controllers/tasks_list_controller.py [hunk 1]
```diff
@@ -194,8 +195,24 @@ def _copySelectedTasks(self):
             app = QtWidgets.QApplication.instance()
             app.clipboard().setText(text)
 
-    def availabilityEditAction(self):
+    def updateActions(self):
         if len(self.view.getSelectedTasks()) == 1:
-            self.editTaskAction.setDisabled(False)
+            self.enableTaskActions()
```
Comment (PKEv) [inline qtodotxt/ui/controllers/tasks_list_controller.py]: but now you disable all actions immediately when you select multiple rows.
As I understand there should be a separate condition for editing tasks

**Comment by oroulet** [comment]
Yes but we can improve later

**Pull request merged**